Modules/_collectionsmodule.c

```diff
@@ -910,7 +910,8 @@ deque_rotate(dequeobject *deque, PyObject *const *args, Py_ssize_t nargs)
 }
 
 PyDoc_STRVAR(rotate_doc,
-"Rotate the deque n steps to the right (default n=1).  If n is negative, rotates left.");
+"Rotate the deque n steps to the right (default n=1). "
+"If n is negative, rotates left.");
```

Comment:
This appears to only be a style change, as far as I can tell. Is it really necessary, or can it be reverted?

```suggestion
"Rotate the deque n steps to the right (default n=1).  If n is negative, rotates left.");
```

Or, if we do want to change this, would we not want to include the signature and format `n` appropriately, as with the others?

```suggestion
"rotate(n)\n\n"
"Rotate the deque *n* steps to the right (default ``n= 1``). "
"If *n* is negative, rotates left.");
```